GetStream/stream-chat-android
# [CAS-938] Create documenation for Client > Moderation Tools section

https://stream-io.atlassian.net/browse/CAS-938

### 🎯 Goal

Create Documentation for "Client > Moderation Tools" section

### 🎨 UI Changes

![screencapture-localhost-3000-docs-client-clientModerationTools-2021-04-29-05_27_33](https://user-images.githubusercontent.com/9600921/116495741-34496a80-a8ac-11eb-96e9-90144efc4910.png)

### 🧪 Testing

1. Go to documentation folder
2. Run `yarn run start`
3. Check out Client -> Moderation Tools section

### ☑️ Checklist


### 🎉 GIF

![giphy (1)](https://user-images.githubusercontent.com/9600921/116486963-8633c580-a897-11eb-9342-6739b47ee9c3.gif)
